ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/PageType.kt

```diff
@@ -54,6 +54,16 @@ enum class PageType(internal val stringValue: String) {
      */
     MINI_CART("mini_cart"),
 
+    /**
+     * The payment method selection page
+     */
+    PAYMENT_METHOD_SELECTION("payment_method_selection"),
```

Comment:
Are we able to add any sort of tests for these? Not sure if we have those currently.

Reply:
It doesn't look like we have tests for this enum.

I feel that testing hardcoded values of enum properties is not worth it as there is no logic. However, if there are functions defined within enums that change based on external input, it'd be worth testing.

Reply:
Gotcha, wasn't sure if we were testing the `page_type` being passed correctly anywhere. It seems like no, but was still worth asking.